pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1173,6 +1173,42 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 	return false
 }
 
+// EnableFSS helps enable feature state switch in the FSS config map
+func (c *K8sOrchestrator) EnableFSS(ctx context.Context, featureName string) error {
+	log := logger.GetLogger(ctx)
+	if c.clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
+		csifeaturestatesconfigmap, err := c.k8sClient.CoreV1().ConfigMaps(cnsconfig.DefaultCSINamespace).Get(ctx,
+			cnsconfig.DefaultSupervisorFSSConfigMapName, metav1.GetOptions{})
+		if err != nil {
+			log.Errorf("failed to "+
+				"get configmap: %q from namespace: %q. err: %v",
+				cnsconfig.DefaultSupervisorFSSConfigMapName, cnsconfig.DefaultCSINamespace, err)
+			return err
+		}
+		csifeaturestatesconfigmap.Data[featureName] = "true"
+		csifeaturestatesconfigmap, err = c.k8sClient.CoreV1().ConfigMaps(cnsconfig.DefaultCSINamespace).Update(ctx,
+			csifeaturestatesconfigmap, metav1.UpdateOptions{})
+		if err != nil {
+			log.Errorf("failed to update configmap: %q in namespace: %q to enable FSS %q. err: %v",
+				cnsconfig.DefaultSupervisorFSSConfigMapName, cnsconfig.DefaultCSINamespace, featureName, err)
+			return err
+		}
+		log.Infof("Successfully updated configmap: %q in namespace: %q to enable FSS %q. ConfigMapData: %v",
+			cnsconfig.DefaultSupervisorFSSConfigMapName, cnsconfig.DefaultCSINamespace, featureName,
+			csifeaturestatesconfigmap.Data)
+		return nil
+	} else {
+		return fmt.Errorf("EnableFSS is not implemented for clusterFlavor: %q", c.clusterFlavor)
+	}
+}
+
+// DisableFSS helps disable feature state switch in the FSS config map
+func (c *K8sOrchestrator) DisableFSS(ctx context.Context, featureName string) error {
+	log := logger.GetLogger(ctx)
+	log.Infof("DisableFSS is not implemented")
+	return nil
```

Comment:
No you want to return Unimplemented error code with error message instead of logging info message?

Reply:
done.

```golang
// DisableFSS helps disable feature state switch in the FSS config map
func (c *K8sOrchestrator) DisableFSS(ctx context.Context, featureName string) error {
	log := logger.GetLogger(ctx)
	return logger.LogNewErrorCode(log, codes.Unimplemented,
		"DisableFSS is not implemented.")
}
```